### PR TITLE
Added keyval_to_unicode

### PR DESCRIPTION
**Pull request by trolleyman**
So yeah... Back again since I accidentally cargo updated a few minutes ago... This ok?

### CODE REVIEW

File: src/keys.rs [hunk 1]
```diff
@@ -7,9 +7,21 @@
...
+use std::mem::transmute;
 
 pub fn keyval_name(keyval: u32) -> Option<String> {
     unsafe {
         from_glib_none(ffi::gdk_keyval_name(keyval as c_uint))
     }
 }
+
+pub fn keyval_to_unicode(keyval: u32) -> Option<char> {
+	unsafe {
```
Comment (gkoz) [inline src/keys.rs]: There still seems to be a problem with the indentation -- six spaces instead of four.

Comment (trolleyman) [inline src/keys.rs]: Ah give me a sec

**Comment by gkoz** [comment]
Um, do you still lack the means to squash the commits?

**Comment by GuillaumeGomez** [comment]
@trolleyman: Thanks for your PR ! As @gkoz, can you squash your commits please ?

**Comment by trolleyman** [comment]
Oh, should have it all good now, squashed. Turns out vim was not playing nicely on windows, so I had to switch it out to Sublime Text.

**Comment by gkoz** [comment]
Seems good now.

**Comment by GuillaumeGomez** [comment]
Perfect, I merge.

**Pull request merged**